pages/establishment/pdf/views/establishment.jsx

```diff
@@ -0,0 +1,191 @@
+              <div className="purple-inset">
+                <ReactMarkdown>{establishment.conditions}</ReactMarkdown>
+              </div>
+            </Fragment>
+          ) : (
+            <p>None.</p>
+          )
+        }
+      </section>
+
+      <section className="authorisations break">
+        <h2>Authorisations</h2>
+        {
+          hasAuthorisations ? (
+            <Fragment>
+              {
+                !!killing.length &&
+                  <Fragment>
+                    <h3>Methods of killing not specified in Schedule 1</h3>
+                    {
+                      killing.map(({ method, description }, i) => (
+                        <div className="purple-inset" key={i}>
+                          <h4>{`Method ${i + 1}`}</h4>
+                          <p>{ method }</p>
+                          <h4>Type or description of the animals to which the killing method will be applied</h4>
+                          <p>{ description }</p>
+                        </div>
+                      ))
+                    }
+                  </Fragment>
+              }
+              {
+                !!rehomes.length &&
+                  <Fragment>
+                    <h3 className="rehomes">Setting free and rehoming of protected animals after completion of a series of regulated procedures</h3>
+                    {
+                      rehomes.map(({ method, description }, i) => (
+                        <div className="purple-inset" key={i}>
+                          <h4>Circumstances under which this may occur</h4>
+                          <p>{ method }</p>
+                          <h4>Animals to be set free or re-homed</h4>
+                          <p>{ description }</p>
+                        </div>
+                      ))
+                    }
+                  </Fragment>
+              }
+            </Fragment>
+          ) : (
+            <p>None.</p>
+          )
+        }
+      </section>
+
+      {/* page break */}
+
+      {
+        hasApprovedAreas ? (
+          <Fragment>
+            {
+              map(establishment.places, (places, siteName) => (
+                <section className="approved-areas break" key={siteName}>
+                  <h2>{`Approved areas ${continued()}`}</h2>
+                  <table>
+                    <thead>
+                      <tr>
+                        <th className="site" colSpan="6">{siteName}</th>
+                      </tr>
+                      <tr>
+                        <th>Area</th>
+                        <th>Name</th>
+                        <th>Suitability</th>
+                        <th>Holding</th>
+                        <th>NACWO</th>
+                        <th>Restrictions</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      {
+                        places.map(place => (
```

Comment:
There's a desire that these are secondarily sorted by `area` and `name` - the API will give them back in database order.

Reply:
I'm adding a sort=site param when I fetch them, do we support secondary sort orders in the api? Better adding it there if not.

Reply:
Yeah, this desire was reported as part of the digital view, so I'm inclined to agree that the API should probably apply those secondary sorts. 

But, a) that's not been done yet, and b) we'd still probably need to make sure that the sort order was preserved once it went through the `groupBy`

Reply:
I've created a ticket for doing the secondary sort in the API.